Makefile

```diff
@@ -151,6 +164,11 @@ kubectl-gadget-container:
 	docker buildx build -t kubectl-gadget -f Dockerfiles/kubectl-gadget.Dockerfile \
```

Comment:
mmm, don't you have to remove `buildx` to accomplish this PR main purpose: "Use docker build rather than buildx to build container image"?

Reply:
Please update the PR description too as it's rather confusing.